src/components/PageBlocks/ChartBase.vue

```diff
@@ -35,23 +37,27 @@ export default {
   },
 
   mounted () {
-    const { chartID } = this.options
-
-    if (chartID === NoID) {
-      return
-    }
-
-    const { namespaceID } = this.namespace
-    this.findChartByID({ chartID, namespaceID }).then((chart) => {
+    this.findChart().then((chart) => {
```

Comment:
How about if instead of the findChart method, you create a fetchChart that finds it and saves it to this.chart. And then you can just call that method in mounted and refresh